packages/manager/modules/hub/src/e2eTests/hubHome/CA_EU_US_hub.e2e.js

```diff
@@ -0,0 +1,54 @@
+import config from '../../../../../tools/testcafe/config';
+import {
+  userRole,
+  userRoleDisconnect,
+} from '../../../../../tools/testcafe/roles/index';
+import HubPage from '../../../../../tools/testcafe/pages/hub/hubPage';
+import OrdersList from '../../../../../tools/testcafe/pages/billing/orders';
+import ServersListDashboard from '../../../../../tools/testcafe/pages/hub/dedicatedServerDashboard';
+import BillsList from '../../../../../tools/testcafe/pages/billing/bills';
+
+const user = userRole(config);
+
+fixture('check hub page')
+  .meta({
+    team: 'BU.Ops.SU.DigitalTools.Product',
```

Comment:
looks like internal informations to me ...

Reply:
seen with @lizardK, it's fine you can keep the meta

Reply:
Is a `service` wouldn't be enough as listed below?

According to a service, we already have to information regarding which team is concerned, right?

https://ovh.github.io/manager/guide/test-e2e.html#documentation

Reply:
yep, can be removed